omnibus/config/software/datadog-agent.rb

```diff
@@ -105,7 +108,7 @@
     mkdir Omnibus::Config.package_dir() unless Dir.exists?(Omnibus::Config.package_dir())
   end
 
-  block do
+  if windows_target? or (not bundled_agents.include? "trace-agent")
     # defer compilation step in a block to allow getting the project's build version, which is populated
     # only once the software that the project takes its version from (i.e. `datadog-agent`) has finished building
```

Comment:
IIUC this is not in a block anymore. Does that mean this comment is not relevant anymore?

Reply:
Removed